docs/en/deploy/Running-Alluxio-On-a-Cluster.md

```diff
@@ -12,26 +12,28 @@ priority: 2
 ## Overview
 
 This section describes the basic setup to run Alluxio with a single master in a cluster.
-This is the simplest way to deploy Alluxio on a cluster, meanwhile this single
-master may also become the single point of failure (SPOF) in an Alluxio cluster. If that machine or
-process became unavailable, the cluster as a whole would be unavailable.
+This is the simplest way to deploy Alluxio on a cluster, meanwhile this single master may also 
+become the single point of failure (SPOF) in an Alluxio cluster.
+If that machine or process became unavailable, the cluster as a whole would be unavailable.
 To deploy Alluxio in production, we highly recommend running Alluxio masters in
 [High Availability]({{ '/en/deploy/Running-Alluxio-On-a-HA-Cluster.html' | relativize_url }}) mode.
 
 ## Prerequisites
 
-* To deploy a Alluxio cluster, first [download](https://alluxio.io/download) the pre-compiled Alluxio
-  binary file, extract the tarball with the below command, and copy the extracted directory to all
-  nodes (including nodes running masters and workers).
+* To deploy a Alluxio cluster, first [download](https://alluxio.io/download) the pre-compiled
+  Alluxio binary file, extract the tarball with the below command, and copy the extracted 
+  directory to all nodes (including nodes running masters and workers).
   
   ```console
   $ tar -xvzpf alluxio-{{site.ALLUXIO_VERSION_STRING}}-bin.tar.gz
   ```
   
-* Enable SSH login without password from master node to worker nodes. You can add a public SSH key for
-  the host into `~/.ssh/authorized_keys`. See [this tutorial](http://www.linuxproblem.org/art_9.html) for more details.
-* TCP traffic across all nodes is allowed. For basic functionality make sure RPC port (default :19998) is open
-  on all nodes.
+* Enable SSH login without password from master node to worker nodes.
+  You can add a public SSH key for the host into `~/.ssh/authorized_keys`.
+  See [this tutorial] (http://www.linuxproblem.org/art_9.html) for more details.
+* TCP traffic across all nodes is allowed.
+  For basic functionality make sure RPC port (default :19998) is open on all nodes.
+* Allow `sudo` privilege for the OS user that Alluxio will be running as.
```

Comment:
why is this a requirement?

Reply:
Further down we mention starting Alluxio with `./bin/alluxio-start.sh all SudoMount`. For consistency, I think we can put this as a requirement. Alternatively, I can add more to the start section to explain how to avoid `sudo`.

Reply:
I don't think it's a hard requirement especially because it depends on configuration and mounting (NVMe?). `sudo` is only ever required to actually make the mount which is separate from Alluxio functionality. I think it is ok to put that `sudo` is required if users want our scripts to mount the workers' storage directories for them

Reply:
done